packages/permissionless/accounts/etherspot/utils/signMessage.ts

```diff
@@ -0,0 +1,39 @@
+import {
+    type Account,
+    type Chain,
+    type Client,
+    type LocalAccount,
+    type SignMessageParameters,
+    type SignMessageReturnType,
+    type Transport,
+    hashMessage,
+    publicActions
+} from "viem"
+import { signMessage as _signMessage } from "viem/actions"
+import { type WrapMessageHashParams, wrapMessageHash } from "./wrapMessageHash"
+
+export async function signMessage<
+    TChain extends Chain | undefined,
+    TAccount extends Account | undefined
+>(
+    client: Client<Transport, TChain, TAccount>,
+    {
+        account: account_ = client.account,
+        message,
+        accountAddress
+    }: SignMessageParameters<TAccount> & WrapMessageHashParams
+): Promise<SignMessageReturnType> {
```

Comment:
can we make chainId an explicit parameter, since we do make a chain id call in the account, this will save an extra chain id call if client doens't have chain. Checkout `GetChainParameter<TChain, TChainOverride>` in https://github.com/pimlicolabs/permissionless.js/blob/main/packages/permissionless/experimental/eip7677/actions/getPaymasterData.ts#L100

Reply:
Changes done.